package.json

```
@@ -32,6 +32,7 @@
     "mocha": "cross-env NODE_ENV=test mocha --exit test/ --recursive",
     "mocha-inspect": "cross-env NODE_ENV=test mocha --exit --inspect-brk --inspect test/ --recursive",
     "migration": "migrate",
+    "migration:sync": "migrate list --autosync",
```

Should we stick to the dash for migration-related tasks?
```suggestion
    "migration-sync": "migrate list --autosync",
```